tests/queries/0_stateless/01849_degrees_to_s2.sql

```diff
@@ -0,0 +1,4 @@
+select degreesToS2(55.77922738, 37.63098076);
```

Comment:
It'll be nice to expand test cases by generating more random coordinates using some kind of tool. for example, S3 has a tool that helps you do this: https://github.com/ClickHouse-Extras/h3/blob/master/src/apps/testapps/mkRandGeo.c